server/config/config.go

```diff
@@ -212,8 +192,7 @@ type Config struct {
 	GRPC      GRPCConfig       `mapstructure:"grpc"`
 	GRPCWeb   GRPCWebConfig    `mapstructure:"grpc-web"`
 	StateSync StateSyncConfig  `mapstructure:"state-sync"`
-	Store     StoreConfig      `mapstructure:"store"`
-	Streamers StreamersConfig  `mapstructure:"streamers"`
+	Streaming StreamingConfig  `mapstructure:"streaming"`
```

Comment:
A follow-up, or this PR needs to update with how look like the new configuration: https://github.com/cosmos/cosmos-sdk/blob/main/tools/confix/data/v0.48-app.toml
Possibly, we need to add something in the UPGRADING.md as well.

Reply:
I updated v048-app.tom. 

I'm certain on the process of UPGRADING.md but if you make a suggestion I can commit it.